src/scripts/release.zig

```diff
@@ -153,10 +153,16 @@ fn build_tigerbeetle(shell: *Shell, info: VersionInfo, dist_dir: std.fs.Dir) !vo
     log.info("building tigerbeetle", .{});
     try shell.project_root.setAsCwd();
 
-    const llvm_lipo_version = shell.exec_stdout("llvm-lipo -version", .{}) catch {
+    const llvm_lipo = for (@as([2][]const u8, .{ "llvm-lipo-16", "llvm-lipo" })) |llvm_lipo| {
```

Comment:
This tool is known as `llvm-lipo-$LLVM_VERSION` on ubunty, and just `llvm-lipo` for me locally. Wrote the code to try both for convenience.